### DISCUSSION TITLE
how to run tests with multiple environments?

### DISCUSSION THREAD

**Discussion by dreamworlds1234** [AnWeber/vscode-httpyac#256] [Q&A]
In this wiki:

https://httpyac.github.io/guide/environments.html

it says on the TIP:

TIP
It is possible to select several environments at the same time.

When I select multiple environments, it seems only run a single environment in the vscode "TEST EXPLORER", regardless the play button or "send" button.  

How do have the TEST run all the environments listed in httpyac.config.js?

Thanks

**Comment by AnWeber** [comment]
@dreamworlds1234 I have just tested the feature and it works as expected. You have to select the appropriate environments in vscode and these are also used in the test runner (unless you have configured httpyac.testRunAlwaysUseEnv). If you want them to be selected automatically, you would have to configure `httpyac.testRunAlwaysUseEnv`. Then these will always be used.
<img width="589" alt="image" src="https://github.com/AnWeber/vscode-httpyac/assets/13434739/eb7fbb7e-4699-440c-9342-46f75a27152e">
